python/QC4_HV.py

```diff
@@ -21,7 +21,7 @@
 cmd.append("--SelectRowEnd= 37")
 cmd.append("--CanvRangeX= 0,1000")
 cmd.append("--CanvRangeY= 0,7")
-cmd.append("--YaxisScale= True")
+cmd.append("--YaxisScale=true")
```

Comment:
Looking at how you have defined the options menu:

```
parser.add_argument('--YaxisScale',dest="YaxisScale",action="store",type=bool, default=False,help="If YaxisScale option is True the Y axis is plotted in kUnit. Default is: False")
```

Can it be that the action is changes from 'store' to 'store_true'?  In this case then all that is needed is to call the option `--YaxisScale` rather than define the value.

Again what is motivating the choice of `argparse` over `optparser`?